include/measurement_kit/report/entry.hpp

```diff
@@ -13,7 +13,7 @@ namespace report {
 // A report entry. This is implemented using private inheritance from
 // nlohmann/json such that we are not exposing externally we are using
 // such json library to implement this class.
-class Entry : private nlohmann::json {
+class Entry : public nlohmann::json {
```

Comment:
To open it as public and perhaps to declare `Entry` as alias for `nlohmann::json`.